src/components/CodeInput/inputItem.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledInput } from './styled';
+
+const InputItem = React.forwardRef((props, ref) => {
+    const {
+        value,
+        index,
+        disabled,
+        required,
+        readOnly,
+        error,
+        tabIndex,
+        onClick,
+        onChange,
+        onFocus,
+        onBlur,
+        onKeyDown,
+        onPaste,
+        id,
+        isActive,
+    } = props;
+
+    const handleOnChange = (e, inputIndex) => {
+        onChange(e.target.value, inputIndex);
+    };
+
+    const handleOnKeyDown = (e, inputIndex) => {
+        const shouldMovePrev = !value && e.key === 'Backspace';
```

Comment:
key must be checked by key code and they are use in one place: `src/libs/constants.js`